app/helpers/environment_banner_helper.rb

```diff
@@ -16,6 +16,6 @@ def environment_banner
   end
 
   private def sentry_env
-    ENV.fetch("SENTRY_ENV")
+    ENV.fetch("SENTRY_ENV", "local_development")
```

Comment:
Just wanted to point out a potential bug.

If SENTRY_ENV is absent in any of the live environments, we will see the `LOCAL DEVELOPMENT ENVIRONMENT` banner. This might not be a problem if we are confident that SENTRY_ENV will never be absent

Reply:
+1, possibly we should refactor the banner to use a separate env variable (in a separate ticket)

Reply:
Totally valid! Thanks both. I guess it's a long shot as we have all three live environments now and the `SENTRY_ENV` is set for all of them.

This feels like a card for a proper backlog? As I have the approvals - i'll go do this and link it here before merging.

Reply:
https://trello.com/c/SSBwCdx9